src/Apps/Home/Components/HomeContentCards.tsx

```diff
@@ -5,6 +5,70 @@ import {
   StaticHeroUnit,
 } from "Apps/Home/Components/HomeHeroUnits/HomeHeroUnit"
 import { CaptionedImage as BrazeContentCard } from "@braze/web-sdk"
+import {
+  Box,
+  ResponsiveBox,
+  GridColumns,
+  Column,
+  SkeletonBox,
+  SkeletonText,
+  Spacer,
+} from "@artsy/palette"
+import { Media } from "Utils/Responsive"
+
+const FakeCard = () => {
+  return (
+    <GridColumns bg="black5" width="100%">
+      <Column span={6} bg="black30">
+        <>
+          <Media at="xs">
+            <ResponsiveBox aspectWidth={3} aspectHeight={2} maxWidth="100%" />
+          </Media>
+          <Media greaterThan="xs">
+            <Box height={[300, 400, 500]} position="relative" />
```

Comment:
`position="relative"` not needed

Reply:
Also copy/paste but you're right I see no visual difference when I remove it. 👍